src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css

```diff
@@ -2064,8 +2064,8 @@ footer{
 	background-size: cover;
 	background-repeat: no-repeat;
 	overflow: hidden;
-	padding-top: 205px;
-	padding-bottom: 190px;
+	padding-top: 130px;
+	padding-bottom: 140px;
```

Comment:
This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)

Reply:
I am unsure how this site was intended to react to changing resolution therefore I don't know the right approach to take while debugging.

Reply:
you can show screenshots how does it look like on different resolution in a browser (F12 and Toggle device toolbar)

Reply:
Added screenshots of "mobile device mode" in screenshots section

Reply:
Ok. Different approach is needed

Reply:
screenshots look ok - maybe not perfect, but ok ;)

Reply:
I wasn't happy how it looked on devices over 1200px wide. It required a bit of learning of css but now it looks ok :)